### ISSUE TITLE
Settings page is a mess

### ISSUE THREAD

**Issue by yratof** [krokedil/woocommerce-gateway-klarna#101]
The settings page for klarna needs organisation, i'm not sure why the plugin has gotten this far without someone pointing this out, 

I would highly recommend putting all the areas of the settings into separate tabs using something like this: http://srsgores.github.io/jquery-hash-tabs/#tab1

For example:

- Placing all the countries in separate tabs would save the settings page from being 4000px long.
- Having required pages together would help as things like the terms page, if not populated, breaks the whole checkout process.
- Developer and test mode settings in a other tab
- Colour and customisation settings in a tab

If this isn't something of a priority, please let me know & i'll throw together a pull request as soon as I can so at least highlight the importance of making the plugin easier to managed from a shop managers perspective.

**Comment by forsvunnet** [comment]
We've done something similar for the bring plugin. The code might be useful for reference: https://github.com/drivdigital/bring-fraktguiden-for-woocommerce/commit/ab816b48eea7ce7022f76d08c94439aa64b400fd

**Comment by slobodan** [comment]
@yratof I agree it can be much better. I worked on a solution for another one of our plugins and will probably implement that here as well. Let me know what you think about this. It is similar to your proposed tabs solution, but uses collapsible elements instead, because the page already has tabsception problem.

Btw. we can't just do any arbitrary HTML in this page, it has to use WC_Settings_API, where you just register your fields and WooCommerce keeps spitting them out, as long as that makes the page.

![screen shot 2017-09-04 at 14 52 55](https://user-images.githubusercontent.com/129607/30027341-c7c3969c-9180-11e7-9c5c-224627ea65c6.png)

@forsvunnet can you upload screenshot of Bring settings page?

**Comment by yratof** [comment]
I like where you were heading with the collapses, but that just makes it harder to find the content you're looking for, when you expand something you're looking for but don't find it, you'll most like expand another and another, leaving you with the long page again. I think that it might be possible to tidy that up without using any additional HTML, simply whats available.
